.codeclimate.yml

```diff
@@ -0,0 +1,23 @@
+exclude_patterns:
```

Comment:
Added this file based on conversation with Benjamin in core pod channel: https://bbcnews.slack.com/archives/GHNLB0MMK/p1562057332065500

Reply:
Ah I meant in a separate PR. Also these are the exclude patterns we currently have, let's not add to these without discussion please (on that separate PR)....
config/
db/
dist/
features/
**/node_modules/
script/
**/spec/
**/test/
**/tests/
Tests/
**/vendor/
**/*_test.go
**/*.d.ts
cypress/{integration,support}/*.js
src/**/*.test.{jsx,js}
src/app/lib/config/**

Reply:
Deleted the codeclimate.yml from here - it's now in this PR: https://github.com/bbc/simorgh/pull/2318